libvmi/libvmi_build.py

```diff
@@ -60,6 +60,7 @@ def check_header(header):
 
 includes = []
 includes.extend(glib_includes)
+includes.extend(["/usr/local/include"])
```

Comment:
Harcoding this path doesn't look right to me

Reply:
do you have any better idea? I am not familiar with this build system and just adding the default installation path should not hurt.

Reply:
Normally you want to detect the path that's required based on dependencies. I don't know anything about the python build system though. /cc @Wenzel